vizro-core/src/vizro/static/css/layout.css

```diff
@@ -120,14 +125,12 @@
   align-items: center;
   display: inline-flex;
   flex-direction: column;
-  padding-top: 26px;
-  width: 64px;
 }
 
 .icon-button {
   align-items: center;
   display: flex;
-  height: 64px;
+  height: 80px;
```

Comment:
Is the new height value according to figma designs? The icon looks a bit small compared to the empty space around it.

Reply:
That's a good question! I've consciously deviated from the old Figma design because it didn't make much sense with the current paddings e.g. we have an outer padding of 32px everywhere except for the navigation bar. So the `dashboard-title`or logo always look a bit misaligned with the navigation bar, while they are aligned with the rest of the elements (e.g. when the navigation bar doesn't exist). I think the old figma designs did not consider that, and we really do need designs with all containers on page. 

However, until we get that, I took Kedro Viz as a reference: https://demo.kedro.org/ - Their navigation bar is also 80px wide. This also looks better aligned for us.